app/helpers/datasets_helper.rb

```diff
@@ -85,6 +85,11 @@ def metadata_files(dataset)
     files
   end
 
+  def licence_url_exits?(dataset)
```

Comment:
```suggestion
  def licence_url(dataset)
```

I _think_ this might be a typo of `exists`, but also I think it's better to avoid using the `exists?` part in this name because it implies the return value will be a `true` or `false` rather than the value itself.

Reply:
Thanks @thomasleese that's a good point! Also indeed was a typo @benjamineskola 😬